go.mod

```diff
@@ -3,27 +3,40 @@ module github.com/ChainSafe/sygma-fee-oracle
 go 1.17
 
 require (
+	github.com/aws/aws-sdk-go-v2/config v1.1.1
```

Comment:
We can remove these dependencies (related to aws), but I will open a separate PR for this.